server/public/plugin/supervisor.go

```diff
@@ -27,11 +27,52 @@ type supervisor struct {
 	client      *plugin.Client
 	hooks       Hooks
 	implemented [TotalHooksID]bool
-	pid         int
```

Comment:
Should we keep the `pid`? It seems helpful to have it so the MM server can check things like resource consumption or file locks.

Reply:
2/5, I saw it was completely unused now and probably suggest removing something until we actually start using it. Easy to bring back if we want to monitor resource consumption or file locks in the future.